templates/terraform/examples/service_directory_endpoint_basic.tf.erb

```diff
@@ -0,0 +1,25 @@
+resource "google_service_directory_namespace" "<%= ctx[:primary_resource_id] %>" {
```

Comment:
We should also get handwritten update tests for any resources/fields that can be updated